react/next.js/app/page.js

```diff
@@ -0,0 +1,49 @@
+// Marks the file as a Next.js Client Component, so that it will be rendered on the client
```

Comment:
```suggestion
// Marks the file as a Next.js Client Component, so that it will only be rendered on the client,
// as the TalkJS JavaScript SDK, which the React SDK uses, can only run in the browser.
```